backend/globaleaks/jobs/log_sched.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+#
+#   log_sched
+#   *********
+#
+import os
+from twisted.internet.defer import inlineCallbacks, returnValue
+from storm.expr import Desc
+
+from globaleaks.jobs.base import GLJob
+from globaleaks.settings import transact, transact_ro
+from globaleaks.models import Log
+from globaleaks.utils.utility import log
+from globaleaks.utils.logger import LogQueue, LoggedEvent, initialize_LoggedEvent
+
+
+class LogSchedule(GLJob):
+    """
+    This class just flush the logs in the Database, is executed quite
+    frequently, and the goal is to record the event safely. the events
+    are kept in memory, by the globaleaks.utils.logger.Log* classes
+
+    """
+    name = "Log"
+
+    highest_logged_id = 0
+
+    @transact_ro
+    def initialize_highest_id(self, store):
+
+        max_id_check = store.find(Log)
+        max_id_check.order_by(Desc(Log.id))
+
+        if max_id_check.count() == 0:
+            log.debug("First initialization spotted!")
+            LogSchedule.highest_logged_id = 1
+        else:
+            LogSchedule.highest_logged_id = max_id_check[0].id
+
```

Comment:
the correct way to do this is simply: store.find(Log).max(Log.id)

check what is the  result when the table is empy and use it accordingly.